cards/src/main/resources/cards/custom/Soulseeker/setless/minion_covetous_soul.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Covetous Soul",
+  "baseManaCost": 6,
+  "type": "MINION",
+  "heroClass": "DARKBLUE",
+  "baseAttack": 4,
+  "baseHp": 3,
+  "rarity": "EPIC",
+  "race": "SPIRIT",
+  "description": "Aftermath: Take control of a random enemy minion.",

```

Comment:
this effect is way too powerful for its cost, not even counting deathrattle triggering effects